playwright/api/basics.ts

```diff
@@ -50,7 +50,17 @@ export class BasicApiService {
       data: { query },
     });
     const loginResponseJson = await loginResponse.json();
+    const errors = loginResponseJson.data.tokenCreate.errors;
 
-    return loginResponseJson as ApiResponse<TokenCreateResponse>;
+    if (
+      test.info().title ===
+      "TC: SALEOR_137 Admin User should be able to deactivate other user @e2e @staff-members"
+    ) {
+      await expect(errors[0].code).toEqual("INACTIVE");
+    } else {
+      await expect(errors).toEqual([]);
+    }
```

Comment:
Why do you add assertions here and not in a test that uses the `logInUserViaApi` function?

Reply:
You're right - I moved it to auth.setup.ts